webui/react/src/components/TableBatch.module.scss

```diff
@@ -1,8 +1,10 @@
 .base {
   height: 0;
   opacity: 0;
+  pointer-events: none;
   position: relative;
   transition: height 0.2s ease-in-out, opacity 0.2s ease-in-out;
+  z-index: 3;
```

Comment:
Why 3 and not 2? (the only other z-index in this file is 1).

If that's due to another file z-index we should probably use variables for z-index values so that we can understand the related logic. The reasoning behind this is simply that is hard to debug z-index issues (or find the right value for a new one) if those are spread among different files.. With variables we could keep them close.

Reply:
the variable approach makes sense to me. I'll add this to the webUI AG topic